CONTRIBUTING.md

```diff
@@ -0,0 +1,28 @@
+## Contributing to Gnocchi
+
+### Issues
+
+We use the [Gihub issue tracker](https://github.com/gnocchixyz/gnocchi/issues) for reporting issues. Before opening a new issue, ensure the bug was not already reported by searching on Issue tracker first.
+
+If you're unable to find an open issue addressing the problem, open a new one. Be sure to include a title and clear description, as much relevant information as possible, and a code sample or an executable test case demonstrating the expected behavior that is not occurring.
+
+
+### Contributing to Documentation
+
+When contributing to the documentation, please add `[ci skip]` to your documentation commit message to skip CI build on that commit. Please remember to use it for commits containing only documentation changes.
```

Comment:
We actually don't want to skip the CI on doc update. We test various things since the web site is built from that.